proposals/nnnn-static-icon-capability.md

```diff
@@ -0,0 +1,52 @@
+# Static Icon Capability
+
+* Proposal: [SDL-NNNN](NNNN-static-icon-capability.md)
+* Author: [Joel Fischer](https://github.com/joeljfischer)
+* Status: **Awaiting review**
+* Impacted Platforms: [RPC / HMI]
+
+## Introduction
+
+Add a capability for app developers to know if static icons are available on a connected head unit.
+
+## Motivation
+
+Currently, any app developer building their app with static icons have no way to know if a connected head unit supports those static icons. We should add a capability for the developer to know if static icons exist or not.
```

Comment:
Change to: 
Currently, any app developer building their app with static icons has no way to know if a connected head unit supports those static icons. We should add a capability for the developer to know if static icons exist or not.